force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,17 +417,29 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
+        Map<String,Integer> empSalaries = new Map<String,Integer>();
 
         // Add key-value pairs to the Map
         // add John Doe => 50000
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
+        empSalaries.put('John Doe', 50000);
+        empSalaries.put('Jane Smith', 60000);
+        empSalaries.put('Sam Brown', 55000);
+        empSalaries.put('Alice Johnson', 65000);
 
         // Get the salary of the employee
-
+        Integer empSalary;
+        for(String key : empSalaries.keySet()){
```

Comment:
You dont need to loop here and check, get for the key will take the one you are looking for without checking all of the keys first. `empSalary = empSalaries.get(employeeName);` is all you need